resalloc_openstack/delete/main.py

```diff
@@ -27,16 +27,17 @@
 from keystoneauth1.exceptions import ConnectionError
 
 def gather_tasks(gc):
+    server = None
     args = parser.parse_args()
 
     try:
         server = Server(nova, args.name)
+        gc.add('10_server', server)
```

Comment:
There are magic values such as this one `10_server` and later down `05_volume`. Can you please move them out as constant and document their value?

Reply:
These magic values are just to let the `gc` object remove them in the right (alphabetical order), no matter in which order the items are added.

Reply:
Ah, that wasn't clear at first sight, and may be worth adding some comment to it. I thought they are values that you found somewhere in our OpenStack instance.

Reply:
Good point, I've documented the `GarbageCollector.add()` method, can you take one more look?

Reply:
Perfect, it makes a lot of sense now.
+1